app/controllers/iep_documents_controller.rb

```diff
@@ -0,0 +1,37 @@
+class IepDocumentsController < ApplicationController
+
+  before_action :find_iep_document, :authorize!
```

Comment:
I wanted to share the `@iep_document` instance variable between the `authorize!` method and the `show` method. Open to improvements!